### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by IshmaelKargbo**
Please review my branch and give you a comment.
commit includes:
1. Contact section
2. Footer section

Thanks 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ Ishmael portfolio is a website where I showcase my professional skills and proje
 - Twitter: [@IshoKargbo](https://twitter.com/IshoKargbo)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/ishmael-kargbo-9a986a214)
```
Comment (SajjadAhmad14) [inline README.md:23]: Please add instructions to run the project locally here.

File: index.html [hunk 2]
```diff
@@ -226,6 +226,60 @@ <h1>Skills</h1>
...
+                                <label for="name" class="label">Your name</label>
+                                <input required class="input" maxlength="30" type="text" name="name" id="name" placeholder="Enter your name">
+                            </li>
+                            <li class="form-label">
+                                <label for="email" class="label">Your name</label>
+                                <input required class="input" type="email" name="email" id="email" placeholder="Enter your email">
+                            </li>
+                            <li class="form-label">
+                                <label for="msg">Message:</label>
+                                <textarea maxlength="500" required class="input" rows="3" name="name" id="msg" placeholder="Enter your name"></textarea>
```
Comment (SajjadAhmad14) [inline index.html:251]: Please use descriptive text for placeholder. It should not be `Enter your name`.

File: index.html [hunk 3]
```diff
@@ -226,6 +226,60 @@ <h1>Skills</h1>
                 </div>
```
Comment (SajjadAhmad14) [inline index.html:226]: Please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ishmael Kargbo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (IshmaelKargbo) [reply index.html:226]: I don't seem to understand this

File: index.html [hunk 4]
```diff
@@ -226,6 +226,60 @@ <h1>Skills</h1>
...
             </div>
         </section>
```
Comment (uwadonat) [inline index.html:228]: - [ ] Let's try to work on all details of the project to be more professional. [This](https://github.com/microverseinc/linters-config/tree/master/html-css) includes the files shown below.
Screenshot from 2021-09-28 20-24-23

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@IshmaelKargbo** , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
strong



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by uwadonat — APPROVED** [review]
Hi **@IshmaelKargbo** , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
cong1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**